assets/js/base/components/radio-control/style.scss

```diff
@@ -97,6 +97,11 @@
 				background: $input-text-dark;
 			}
 		}
+
+		&[disabled] {
```

Comment:
Should we also set `pointer-events: none` here? Not sure if it's really needed if the inputs are disabled

Reply:
We're applying the "cancel" cursor to indicate it's not clickable when disabled. I think pointer events would disable that but maybe thats a cleaner solution.

Reply:
I tested this and it just leaves the default pointer. Nothing else changes, and its already not clickable because of the disabled attribute. I'll leave it with the not-allowed cursor.